eating_cookies/eating_cookies.py

```diff
@@ -6,7 +6,13 @@
 # a solution that is more efficient than the naive 
 # recursive solution
 def eating_cookies(n, cache=None):
-  pass
+  if n < 0:
+    return 0
+  elif n == 0:
+    return 1
+
```

Comment:
I see what you're doing with cache, but I think you may be missing an elif or two